config/initializers/assets.rb

```diff
@@ -26,6 +26,4 @@
 Rails.application.config.assets.precompile += %w( potlatch2/FontLibrary.swf )
 Rails.application.config.assets.precompile += %w( potlatch2/locales/*.swf )
 Rails.application.config.assets.precompile += %w( help/introduction.* )
-Rails.application.config.assets.precompile += %w( iD/img/*.svg iD/img/*.png iD/img/*.gif )
-Rails.application.config.assets.precompile += %w( iD/img/pattern/*.png )
-Rails.application.config.assets.precompile += %w( iD/locales/*.json )
```

Comment:
It's supposed to just precompile everything under in `/vendor/assets/iD/`  I felt that would be easier than listing every subfolder and wildcarding the files because this new traffico stuff contains a bunch of subfolders.  Some icon fonts, some css, a lot of json files.